UnityProject/Assets/Editor/LevelBuilder/Generators/TerrainGenerator.cs

```diff
@@ -0,0 +1,29 @@
+using UnityEngine;
+using UnityEditor;
+
+public class TerrainGenerator
+{
+    private const float TerrainScalingFactor = 0.1f;
+    private const float TerrainSnapToGridShift = -0.5f;
+    private GameObject terrainGameObject;

```

Comment:
what are the coding conventions here for naming?

Reply:
I don't think we have agreed to any... Unity and Microsoft have slightly different conventions. I think it's worth creating an issue for

Reply:
#80